src/samplers/usercall/config.rs

```diff
@@ -0,0 +1,79 @@
+// Copyright 2019-2021 Twitter, Inc.
```

Comment:
Please use just `2021` we have decided against using year ranges. (I may need to cleanup existing files)